docs/scheduled-transactions/revised-spec.md

```diff
@@ -0,0 +1,157 @@
+# Scheduled transactions 
+
+An ordinary transaction must be submitted to the network with the signatures 
+of enough Ed25519 keys to activate all the Hedera keys required to sign it. 
+Otherwise it will resolve to `INVALID_SIGNATURE`. 
+
+Here we describe a new kind of _scheduled_ transaction that is not directly 
+submitted to the network, but rather created as part of a _schedule entity_ in 
+the network's action queue. An Ed25519 key "signs" a scheduled transaction
+by signing an ordinary transaction that either creates or affirms the 
+schedule entity.
+
+Along with a scheduled transaction, a schedule entity (or simply _schedule_) also contains,
+  1. An optional memo. 
+  2. An optional admin key that can be used to delete the schedule.
+  3. An optional account to be charged the service fee for the scheduled transaction.
+  4. A list of the Ed25519 keys that the network deems to have signed the scheduled transaction.
+  5. A flag that says if the scheduled transaction may be executed as soon as the schedule has collected enough signing keys. 
```

Comment:
I thought we said we would remove all non-implemented fields like `executeImmediatelyOnValidSignatures`?

Reply:
This still needs to be removed

Reply:
Removed.